bedrock/firefox/templates/firefox/enterprise/sla.html

```diff
@@ -0,0 +1,615 @@
+                <p>{{ _('Operation issue - Firefox cannot be installed or opened on 5% or less of the Enterprise computers or cannot access 5% or less of ALEXA 500 websites') }}</p>
+              </td>
+            </tr>
+            <tr class="t-severity-light-green">
+              <th>
+                <h4>{{ _('Severity 3 Management') }}</h4>
+                <p>{{ _('Issues that are not critical to Firefox stability, security or compatibility with websites on the Enterprise current deployment.') }}</p>
+                <p>{{ _('Ticket types that can be severity 3: Support request on customization or deployment') }}</p>
+              </th>
+              <td>
+                <p>{{ _('Standard support how-to requests to use various Enterprise or other features.') }}</p>
+              </td>
+            </tr>
+            <tr class="t-severity-green">
+              <th>
+                <h4>{{ _('Severity 4 Feature request') }}</h4>
+                <p>{{ _('Customer requests for new features. Ticket will be resolved with a reference to an opened Bugzilla ticket for later action.') }}</p>
+                <p>{{ _('Ticket types that can be severity 4: Feature request') }}</p>
+              </th>
+              <td>
+                <p>{{ _('Request to include a new feature into the next release of Firefox') }}</p>
+              </td>
+            </tr>
+          </tbody>
+        </table>
+      </section>
+
+      <section id="services-hardware" class="mzp-c-details">
+        <h3>{{ _('Hardware Supported') }}</h3>
+        <p>
+          {% trans requirements=url('firefox.sysreq') %}
+          Mozilla will only provide support to those machines that meet our system requirements for the latest supported Firefox releases. Examples of Firefox system requirements can be found <a rel="noopener" target="_blank" href="{{ requirements }} ">here</a>.
+          {% endtrans %}
+        </p>
+      </section>
+
+      <section id="services-configuration" class="mzp-c-details">
+        <h3>{{ _('Configuration Management') }}</h3>
+
+        <p>
+        {% trans %}
+          In order to successfully deploy Firefox for Enterprise, your environment will need some form of configuration management solution that can deploy software packages to the machines that you wish to manage.
+        {% endtrans %}
+        </p>
+
+        <p>
+        {% trans %}
+          Software solutions such as Microsoft Intune, System Center Configuration Manager, JAMF’s JAMF Pro, Chef and Puppet are common configuration management platforms that will allow you to deploy Firefox for Enterprise to the portion of your fleet that will be participating in the pilot, and push updated binaries as they are required.
+        {% endtrans %}
+        </p>
+
+        <p>
+        {# TODO: link to PDF. #}
+        {% trans deployment='https://docs.google.com/document/d/1d5DUVkyKfICzC2t2aNAZ9vFpt4Xhe_yvEV23UxXcJjw/' %}
```

Comment:
Note: we're getting a PDF to link to in place of this google doc, so this PR is blocked until we have that file (it'll live on assets.mozilla.net).